tinywl/tinywl.zig

```diff
@@ -355,9 +355,7 @@ const Server = struct {
             // Focus the next view in the stack, pushing the current top to the back
             .F1 => {
                 if (server.views.length() < 2) return true;
-                const view = @fieldParentPtr(View, "link", server.views.link.next.?);
-                view.link.remove();
-                server.views.append(view);
+                const view = @fieldParentPtr(View, "link", server.views.link.prev.?);
```

Comment:
We still need to push the current top of the list to the back, the bug is that we are then focusing that view again instead of focusing the view that was second in the stack.

Reply:
> that was second in the stack.

Why? I thought the idea was to cycle through the views this way:
```
[a b c d] # a is focused
[d a b c] # d is focused
[c d a b] # c is focused
[b c d a] # b is focused
[a b c d] # a is focused
```

Reply:
Ah, missed that you changed that to `link.prev`. That works fine too, although it is a deviation from tinywl.c. Your way is simpler though so LGTM.